packages/stencil/components/stencil.config.ts

```diff
@@ -42,6 +42,7 @@ const config: Config = {
         'osds-icon',
         'osds-input',
         'osds-link',
+        'osds-select',
```

Comment:
should we add `osds-select-option` & `osds-select-group` ?

Reply:
I don't think so